setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="VoA",
+    version="0.0.1",
+    author="Mateusz Przyborowski, Krzysztof Suwada",
+    description="Tools proposed in the paper 'Validation of Association'",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    url="https://arxiv.org/pdf/1904.06519",
+    packages=find_packages(),
+    install_requires=[
```

Comment:
You should specify lowest versions of packages where it will work

Reply:
Versions added.